src/utils/entitySpec.js

```diff
@@ -15,6 +15,24 @@ const toLowerCaseMap = {
   mavencentralsource: NONE
 }
 
+const NPM_WEBSITE = 'npmjs.com'
+const GITHUB_WEBSITE = 'github.com'
+const MAVEN_WEBSITE = 'mvnrepository.com'
+const NUGET_WEBSITE = 'nuget.org'
+const PYPI_WEBSITE = 'pypi.org'
+const RUBYGEM_WEBSITE = 'rubygems.org'
+
+const providerPath = {
+  [NPM_WEBSITE]: 'npm/npmjs/-',
```

Comment:
The namespace is not always going to be `-` here

For example https://www.npmjs.com/package/@types/lodash it will be `@types`